src/cross_validation.jl

```diff
@@ -0,0 +1,188 @@
+using MLBase
+
+# from Taddy (2015, distrom):
+# CV selection rules: both CV1se, which chooses the largest λt with mean
+# OOS deviance no more than one standard error away from minimum, and CVmin,
+# which chooses λt at lowest mean OOS deviance.
+
+function CVmin(oosdevs)
+  cvmeans = mean(oosdevs,2)
+  segCVmin = indmin(cvmeans)
+end
+
+# function CV1se_slow_but_sure(oosdevs)
+#   cvmeans = vec(mean(oosdevs,2))
+#   (mincvmean,segCVmin) = findmin(cvmeans)
+#   cvstds = vec(std(oosdevs,2)) / sqrt(nfolds-1)
+#   cv1se = (mincvmean + cvstds[segCVmin]) - cvmeans
+#   segCV1se = minimum(collect(1:size(cvmeans,1))[cv1se.>=0])
+# end
+
+function CV1se(oosdevs)
+  nλ,nfolds = size(oosdevs)
+  cvmeans = vec(mean(oosdevs,2))
+  (mincvmean,segCVmin) = findmin(cvmeans)
+  mincvstds = std(oosdevs[segCVmin,:]) / sqrt(nfolds-1)
```

Comment:
It would be marginally efficient to a view here (`view(oosdevs, segCVmin, :)` instead of `oosdevs[segCVmin,:]`), but I doubt this actually a performance bottleneck.